### ISSUE TITLE
Add missing SCSS modulo operator (%)

### ISSUE THREAD

**Issue by hcomnetworkers** [racodond/sonar-css-plugin#29]
```
$component-space: 21px;

@function components-add-margins($bot, $top) {
  // Parameter validation
  @if unit($bot) != "px" or unit($top) != "px" {
    @error "Parameter $bot and $top are expected to be of type 'px'.";
  }

  $sum : ($top + $bot) % $component-space;
  @if $sum <= 0 {
    $sum : $sum + $component-space;
  }
  @return $sum;
}
```

This error is displayed for the "%" symbol:

> Parse error at line 27 column 25: 17: /// 18: /// @param {Number} $bot - Margin in pixels 19: /// @param {Number} $top - Margin in pixels 20: /// @returns {Number} In pixels 21: @function components-add-margins($bot, $top) { 22: // Parameter validation 23: @if unit($bot) != "px" or unit($top) != "px" { 24: @error "Parameter $bot and $top are expected to be of type 'px'."; 25: } 26: 27: $sum : ($top + $bot) % $component-space; ^ 28: @if $sum <= 0 { 29: $sum : $sum + $component-space; 30: } 31: @return $sum; 32: } 33: 34: /// Substracts a margin from the standard margin and takes care that it not results less or equal 0px 35: /// 36: /// This function can handle $sub to be too large or to be negative. 37: /// @example

Sass has no problem with this syntax.

Thank you

**Comment by racodond** [comment]
Hi @hcomnetworkers,

Thanks for your feedback!
I reproduced the error and will fix it in version 4.5.

David

**Comment by racodond** [comment]
Version 4.5 is now available at https://github.com/racodond/sonar-css-plugin/releases/tag/4.5
